client/src/modules/home/index.tsx

```diff
@@ -41,15 +41,26 @@ const Welcome = ({ user }: { user: User }) => {
   );
 };
 const Home = () => {
-  const { error: chapterError, data: chapterData } = useChaptersQuery();
+  const [eventsLoaded, setEventsLoaded] = useState(false);
+  const [getChapters, { error: chapterErrors, data: chapterDatas }] =
+    useChaptersLazyQuery();
 
   const { loading, error, data, fetchMore } = usePaginatedEventsWithTotalQuery({
     variables: { offset: 0, limit: eventsPerPage },
+    onCompleted: () => {
+      setEventsLoaded(true);
+    },
   });
   const { user } = useUser();
 
-  const isLoading = loading || !data || !chapterData;
-  if (isLoading) return <Loading error={error || chapterError} />;
+  useEffect(() => {
+    if (eventsLoaded && !chapterDatas) {
+      getChapters();
+    }
+  }, [eventsLoaded]);
```

Comment:
```suggestion
    getChapters();
  }, [loading]);
```

Reply:
I might have glitched :thinking:. This, double checked, should start loading chapters after events are loaded:
```suggestion
    if (!loading) {
      getChapters();
    }
  }, [loading]);
```